### PR TITLE
Only show permission options for real users

### PR DESCRIPTION
**Pull request by nickvergessen**
1. Add a group to a conversation
2. Open the options menu
3. See permissions things which do not have an effect 😕

### CODE REVIEW

File: src/components/RightSidebar/Participants/ParticipantsList/Participant/Participant.vue [hunk 1]
```diff
@@ -527,6 +527,12 @@ export default {
 			return this.participantTypeIsModerator(this.participantType)
 		},
 
+		canBePermitted() {
```
Comment (marcoambrosini) [inline src/components/RightSidebar/Participants/ParticipantsList/Participant/Participant.vue:530]:
```suggestion
		showPermissions() {
```

**Review by marcoambrosini — APPROVED** [review]

File: src/components/RightSidebar/Participants/ParticipantsList/Participant/Participant.vue [hunk 2]
```diff
@@ -143,7 +143,7 @@
 			</ActionButton>
 
 			<!-- Permissions -->
-			<template v-if="selfIsModerator && !isModerator">
+			<template v-if="selfIsModerator && !isModerator && canBePermitted">
```
Comment (marcoambrosini) [inline src/components/RightSidebar/Participants/ParticipantsList/Participant/Participant.vue:146]:
```suggestion
			<template v-if="showPermissions">
```

File: src/components/RightSidebar/Participants/ParticipantsList/Participant/Participant.vue [hunk 3]
```diff
@@ -527,6 +527,12 @@ export default {
 			return this.participantTypeIsModerator(this.participantType)
 		},
 
+		canBePermitted() {
+			return this.participant.actorType === ATTENDEE.ACTOR_TYPE.USERS
+				|| this.participant.actorType === ATTENDEE.ACTOR_TYPE.GUESTS
+				|| this.participant.actorType === ATTENDEE.ACTOR_TYPE.EMAILS
```
Comment (marcoambrosini) [inline src/components/RightSidebar/Participants/ParticipantsList/Participant/Participant.vue:533]:
```suggestion
			return (this.participant.actorType === ATTENDEE.ACTOR_TYPE.USERS
				|| this.participant.actorType === ATTENDEE.ACTOR_TYPE.GUESTS
				|| this.participant.actorType === ATTENDEE.ACTOR_TYPE.EMAILS)
				&& selfIsModerator
				&& !isModerator
```

File: src/components/RightSidebar/Participants/ParticipantsList/Participant/Participant.vue (hunk 1 above)
Comment (nickvergessen) [reply src/components/RightSidebar/Participants/ParticipantsList/Participant/Participant.vue:530]: `showPermissionsOptions` ?

**Pull request merged**